x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/index.test.tsx

```diff
@@ -128,4 +128,51 @@ describe('NetworkTopNFlow Table Component', () => {
       ).toEqual('by destination ip');
     });
   });
+
+  describe('Sorting on Table', () => {
+    test('when you click on the column header, you should show the sorting icon', () => {
+      const wrapper = mount(
+        <MockedProvider>
+          <TestProviders>
+            <NetworkTopNFlowTable
+              loading={false}
+              data={mockData.NetworkTopNFlow.edges}
+              totalCount={mockData.NetworkTopNFlow.totalCount}
+              hasNextPage={getOr(false, 'hasNextPage', mockData.NetworkTopNFlow.pageInfo)!}
+              nextCursor={getOr(null, 'endCursor.value', mockData.NetworkTopNFlow.pageInfo)!}
+              loadMore={loadMore}
+              startDate={startDate}
+              type={networkModel.NetworkType.page}
+            />
+          </TestProviders>
+        </MockedProvider>
+      );
+
+      wrapper
+        .find('.euiTable thead tr th button')
+        .at(1)
+        .simulate('click');
+
+      wrapper.update();
+
+      expect(
+        wrapper
+          .find('.euiTable thead tr th button')
+          .first()
+          .text()
+      ).toEqual('BytesClick to sort in ascending order');
+      expect(
+        wrapper
+          .find('.euiTable thead tr th button')
+          .at(1)
+          .text()
+      ).toEqual('PacketsClick to sort in descending order');
+      expect(
+        wrapper
+          .find('.euiTable thead tr th button')
+          .at(1)
+          .find('svg')
```

Comment:
I've been using more css styles for selectors with my eui components in my tests lately as well. I am fearful the next time I update from `master` I am going to seeing more broken tests.

No actionable item here other than if you find a `data-test-subj` to use with tables and headers please share them with me so mine aren't as brittle.

The last time @spong and myself updated from `master` there were a lot of CSS changes and (of course) we updated a lot of tests. We ended up using more of a top level `find` and then a `.at(n)` to fix them fwiw.

That seemed like less of breaking changes and easier to fix by changing a `.at(n)` to something like `.at(n - 1)` later than CSS selectors.

Thank you for the tests though. I would rather have tests and fix them occasionally then no tests. 👍

Reply:
I agree but here I do not have that much control since it is inside of the EUi table